tool/planet/membership.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/gravitational/planet/lib/monitoring"
+
+	"github.com/gravitational/trace"
+	serf "github.com/hashicorp/serf/client"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// startSerfReconciler creates a control loop that periodically attempts to
+// reconcile serf cluster.
+func startSerfReconciler(ctx context.Context, serfConfig *serf.Config) {
+	ticker := time.NewTicker(30 * time.Second)
```

Comment:
I think 30 seconds might be a bit too often for this type of check. I'd probably do it once every 5-10 minutes or so.